shared/common-adapters/markdown/shared.js

```diff
@@ -78,6 +80,21 @@ function createChannelRegex(meta: ?MarkdownMeta): ?RegExp {
   return new RegExp(`^#(${meta.mentionsChannelName.keySeq().join('|')})\\b`)
 }
 
+function createKbfsPathRegex(): ?RegExp {
+  const username = `([a-zA-Z0-9]+_?)+` // from go/kbun/username.go
+  const usernames = `${username}(,${username})*`
```

Comment:
🐑 
```suggestion
  const usernames = `${username}(?:,${username})*`
```

And same thing below so you don't capture this group since you don't have to.